core/modules/block/block.module

```diff
@@ -89,7 +111,7 @@ function block_block_info() {
   foreach ($config_names as $config_name) {
     $block = config_get($config_name);
     $blocks[$block['delta']] = $block;
-    $blocks[$block['delta']]['description'] = !empty($block['description']) ? $block['description'] : t('A reusable custom block.');
+    $blocks[$block['delta']]['description'] = t('A reusable custom block.');
```

Comment:
This is a regression that would cause block descriptions to get lost. This was a recently added feature in https://github.com/backdrop/backdrop-issues/issues/2722